distage/distage-core-api/src/main/scala/izumi/distage/model/definition/dsl/ModuleDefDSL.scala

```diff
@@ -745,31 +768,17 @@ object ModuleDefDSL {
       }
     }
 
-//    def modify[I <: T: Tag](f: T => I): MakeDSLUnnamedAfterFrom[T] = {
-//      addOp(Modify[T](_.map(f)))(new MakeDSLUnnamedAfterFrom[T](_, key))
-//    }
-//
-//    def modifyBy(f: ProviderMagnet[T] => ProviderMagnet[T]): MakeDSLUnnamedAfterFrom[T] = {
-//      addOp(Modify(f))(new MakeDSLUnnamedAfterFrom[T](_, key))
-//    }
-
-  }
-
-  final class MakeNamedDSL[T](
-    override protected val mutableState: SingletonRef,
-    override protected val key: DIKey.IdKey[_],
-  ) extends MakeDSLBase[T, MakeDSLNamedAfterFrom[T]]
-    with MakeDSLMutBase[T] {
-
-    def tagged(tags: BindingTag*): MakeNamedDSL[T] = {
-      addOp(AddTags(tags.toSet)) {
-        new MakeNamedDSL[T](_, key)
-      }
+    def annotateParameter[P]: PartiallyAppliedParameterAnnotation[P, MakeDSLUnnamedAfterFrom[T]] = {
```

Comment:
need to add id to `MakeNamedDSL` & `MakeDSLNamedAfterFrom`

Reply:
Done.